### DISCUSSION TITLE
24/10/31

### DISCUSSION THREAD

**Discussion by gimsoomin** [cloud2scape/.github#17] [회의록]
## 일정

멘토링 시간 11/01 17시 온라인
끝나고 회의 진행후 해산 예정

### 멘토
- [x] db 구성
- [x] 하이브리드 해야될지
- [ ] 운영을 어느정도 경험해야할지
- [ ] 데브옵스 피치못하게 빠졌을때 조율하는 방법
- [x] CI/CD을 프라이빗 저장소로 앤서블 구성하는 것
- [x] SRE 에 대해..

### 교육처
- [x] 정산 언제하는지 대리님 문의

### 찬준
모니터링 로키 리눅스에 구현중(Promtail + Loki + Prometheus + Grafana)

### 나연
아르고 cd, eks 테스트 중이나 키문제로 잘 안되는 중
별개로 아키텍처 구성 시작

### 수민
앤서블 테스트 -> 멘토링에 따라  aws와 온프렘 db 구성 설계 디깅 예정

### 재욱
msa 환경 구성(dev, prod)을 위한 gateway, config server 골조 구현 중
msa로 hello world 띄우면 비즈니스 로직 구현 예정

## 특이사항
도메인 살것인지는 도메인 제공처에서 WAF등 기능추가 되는 경우가 있어서 조사가 필요하다

멘토님한테 노션 문서, 깃허브 구성도 보고 예정
https://www.notion.so/laigasus/12d284a4633c809e8134c32fd8ecd839?pvs=4
https://github.com/orgs/cloud2scape/discussions/13#discussion-7381605

## 기타
![image](https://github.com/user-attachments/assets/759b9875-b9b5-43bf-bded-a1b4cb4f2e64)

**Comment by gimsoomin** [comment]
잘못된 부분 있으면 바로 고쳐주세요

**Comment by gimsoomin** [comment]
정산 시점
11월 금액은 12월 초 결제, 12월 금액은 1월 초 결제 이렇게 진행될텐데 결제 후 증빙자료를 전달할 수 있기에 증빙자료 확보 시 12월 중순, 1월 중순쯤 정산 가능할 것 같습니다.

**Comment by ssedogdoom** [comment]
Argocd해결했습니다
